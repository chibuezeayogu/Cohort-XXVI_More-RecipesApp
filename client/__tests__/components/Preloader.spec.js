import React from 'react';
import expect from 'expect';
import { shallow } from 'enzyme';
import Preloder from '../../Components/Preloder';

describe('<Preloder />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<Preloder />);
    expect(wrapper).toMatchSnapshot();
  });
});
