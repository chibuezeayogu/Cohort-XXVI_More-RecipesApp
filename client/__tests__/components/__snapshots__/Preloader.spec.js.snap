// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Preloder /> should render correctly 1`] = `
ShallowWrapper {
  "length": 1,
  "node": <div
    className="preloader-wrapper big active loader"
  >
    <div
      className="spinner-layer spinner-blue-only"
    >
      <div
        className="circle-clipper left"
      >
        <div
          className="circle"
        />
      </div>
      <div
        className="gap-patch"
      >
        <div
          className="circle"
        />
      </div>
      <div
        className="circle-clipper right"
      >
        <div
          className="circle"
        />
      </div>
    </div>
  </div>,
  "nodes": Array [
    <div
      className="preloader-wrapper big active loader"
    >
      <div
        className="spinner-layer spinner-blue-only"
      >
        <div
          className="circle-clipper left"
        >
          <div
            className="circle"
          />
        </div>
        <div
          className="gap-patch"
        >
          <div
            className="circle"
          />
        </div>
        <div
          className="circle-clipper right"
        >
          <div
            className="circle"
          />
        </div>
      </div>
    </div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Preloader />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {},
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="preloader-wrapper big active loader"
        >
          <div
            className="spinner-layer spinner-blue-only"
          >
            <div
              className="circle-clipper left"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="gap-patch"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="circle-clipper right"
            >
              <div
                className="circle"
              />
            </div>
          </div>
        </div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="preloader-wrapper big active loader"
        >
          <div
            className="spinner-layer spinner-blue-only"
          >
            <div
              className="circle-clipper left"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="gap-patch"
            >
              <div
                className="circle"
              />
            </div>
            <div
              className="circle-clipper right"
            >
              <div
                className="circle"
              />
            </div>
          </div>
        </div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Preloader />,
}
`;
